cmake/configure_files/HydrogenConfig.cmake.in

```diff
@@ -180,13 +202,16 @@ if (_HYDROGEN_HAVE_MPC)
     set(MPC_INCLUDE_DIR "@MPC_INCLUDE_DIR@"
       CACHE FILEPATH "The MPC include directory" FORCE)
   endif ()
-endif (_HYDROGEN_HAVE_MPC)
+endif (HYDROGEN_HAVE_MPC)
 
 include("FindAndVerifyExtendedPrecision")
 
 # LAPACK
+# TODO: This should be cleaned up to better support a consistent view
```

Comment:
Agreed. This should probably be something like `HYDROGEN_BLAS_LIBRARY` and set to `OPENBLAS`